app/src/main/java/com/dessalines/thumbkey/utils/Utils.kt

```diff
@@ -748,18 +748,19 @@ fun autoCapitalizeCheck(ime: IMEService): Boolean {
     val empty = ime.currentInputConnection.getTextBeforeCursor(1, 0).isNullOrEmpty()
 
     // Avoid capitalizing in certain edit boxes
+    // TYPE_NULL indicates no automation desired. Notably fixes termux
     val inputType = ime.currentInputEditorInfo.inputType and (InputType.TYPE_MASK_VARIATION)
     if (listOf(
             InputType.TYPE_TEXT_VARIATION_URI,
             InputType.TYPE_TEXT_VARIATION_PASSWORD,
             InputType.TYPE_TEXT_VARIATION_WEB_PASSWORD,
             InputType.TYPE_TEXT_VARIATION_VISIBLE_PASSWORD,
+            EditorInfo.TYPE_NULL,
         ).contains(inputType)
```

Comment:
This isn't exactly the same logic as what was below. I don't exactly know what TYPE_MASK_VARIATION is, but it wasn't checked in the one below.

Reply:
Honestly no idea what that does, should I add it under listof? google says it's "Mask of bits that determine the variation of the base content class."

I don't know how to test what that would change, that description is incredibly vague for me.

Reply:
Seems like you figured it out.